### DISCUSSION TITLE
Permissions sub folder inside a folder

### DISCUSSION THREAD

**Discussion by PH5HP** [rejetto/hfs#756] [General]
Hi rejetto,

I have inside my vritual folders only folders which are physically present on my harddrive. The problem is that I want to give one folder inside one of the  main folders different permissions. I think that at this time it can't be done.

Could you make this possible?

Kind regards,
Henk - PH5HP

**Comment by rejetto** [comment]
it's already possible.
the *easy* way is to add the "inside" folder in the vfs, under the "parent" folder.
now you can just set permissions to "inside".

the *hard* way is to use masks.

**Reply by PH5HP** [reply]
Hi rejetto,

I want to hide a folder inside a folder. I am not logged in and I tried to do this, but I can still see that folder.

![image](https://github.com/user-attachments/assets/ec5ac480-2899-43c2-98e9-09b1bdc840ad)

**Reply by rejetto** [reply]
First add magazine under amateur, then cqpa under magazines

**Reply by rejetto** [reply]
You can Just drag cqpa when you have magazines

**Reply by PH5HP** [reply]
It works ... TNX ...I will try to remember this...

**Comment by rejetto** [comment]
If you have time, it would be nice if you'd write down here or editing at
 https://github.com/rejetto/hfs/wiki/Permissions
Instructions the way you would have liked to get

**Reply by PH5HP** [reply]
You mean you want me to write instructions how to do this what I just have done?...

**Reply by rejetto** [reply]
Yes, in a general way. 
As it's important to know what you would have considered  clear instructions. Im surely biased.
